apps/condo/domains/common/hooks/useSearch.ts

```diff
@@ -21,5 +21,5 @@ export const useSearch = <F> (loading: boolean): [string, (search: string) => vo
         searchChange(value)
     }
 
-    return [search, handleSearchChange]
+    return [search ?? searchValueFromQuery, handleSearchChange]
```

Comment:
It's better not to use `??` here, because we manage to keep similar code style both on back and front, and back doesn't have TS yet, so `??` can cause problems in some cases.

Reply:
@akarjakin, `??` replaced with `||`